core/tauri/src/window.rs

```diff
@@ -1569,11 +1569,15 @@ impl<R: Runtime> Window<R> {
         return Err(Error::InvokeKey);
       }
       None => {
+        // this specific string is tested against in `core/tests/invoke-key`.
+        // if this error changes then that integration test should be updated accordingly.
+        let error = "received ipc message without a __TAURI_INVOKE_KEY__";
+
         #[cfg(feature = "tracing")]
-        tracing::error!("received ipc message without a __TAURI_INVOKE_KEY__");
+        tracing::error!(error);
 
         #[cfg(not(feature = "tracing"))]
-        eprintln!("received ipc message without a __TAURI_INVOKE_KEY__");
+        eprintln!(error);
```

Comment:
```suggestion
        eprintln!("{}", error);
```
Fixes clippy error https://github.com/tauri-apps/tauri/actions/runs/9381784496/job/25831765618?pr=9983#step:6:635